server/src/domain/asset/asset.service.ts

```diff
@@ -389,18 +389,30 @@ export class AssetService {
   async update(authUser: AuthUserDto, id: string, dto: UpdateAssetDto): Promise<AssetResponseDto> {
     await this.access.requirePermission(authUser, Permission.ASSET_UPDATE, id);
 
-    const { description, ...rest } = dto;
+    const { description, creationDate, latitude, longitude, ...rest } = dto;
     if (description !== undefined) {
       await this.assetRepository.upsertExif({ assetId: id, description });
     }
 
+    if (creationDate !== undefined || latitude !== undefined || longitude !== undefined) {
+      await this.jobRepository.queue({
+        name: JobName.WRITE_EXIF,
+        data: {
+          id: id,
+          creationDate: creationDate,
+          GPSLatitude: latitude,
+          GPSLongitude: longitude,
```

Comment:
We should decide on a naming convention. Either (1) exif entity property names or (2) EXIF tag names. IMO, we should use (2) Exif Tag names, in which case this should be "CreationDate" or "DateTimeOriginal". Also, if the key/value is the same you can use a shorthand version of it `{ creationDate }`

Reply:
Actually, I take it back - lets only use exif tag names in the `handleWriteExif` job itself. Let's use exif entity names here instead. So `latitude`, `longitude`, and `dateTimeOriginal`.

Reply:
So now it's:
Exif entity property names everywhere except inside the `handleWriteExif` where it's exif tag names.

This way when you look at `handleWriteExif` you can see what will be effectivly write inside the xmp file